lesson14/js/main.js

```diff
@@ -0,0 +1,174 @@
+'use strict';
+//Modal
+const menuLists = document.createElement('ul');
+
+function createModalWindow() {
+	const modalWindow = document.createElement('div');
+	const createMenuContainer = document.createElement('div');
+
+	//modal container
+	modalWindow.classList.add('bl_modal');
+	modalWindow.setAttribute('id', 'js-modalContainer');
+
+	//menu button
+	const createMenuBtn = document.createElement('button');
+	createMenuBtn.setAttribute('id', 'js-loadmenuBtn');
+	createMenuBtn.setAttribute('class', 'btn');
+	createMenuBtn.textContent = `MENU`;
+
+	//close button
+	const createCloseBtn = document.createElement('button');
+	createCloseBtn.setAttribute('id', 'js-closeModal');
+	createCloseBtn.setAttribute('class', 'el_closeModalBtn');
+	createCloseBtn.textContent = '×';
+
+	//input area
+
+	const createInputButton = document.createElement('input');
+	createInputButton.type = 'number';
+	createInputButton.placeholder = 'please input number';
+	createInputButton.min = '1';
+	createInputButton.setAttribute('id', 'number');
```

Comment:
IMO
The function is long so the fix is affecting other features. Cut out or use a different method

Reply:
Thank you for your review.
I'm sorry that our response was delayed.

Changed to show / hide with CSS instead of creating modal with JavaScript.


[07206c8](https://github.com/nyattsu72/hanson-frontend-cource/pull/19/commits/07206c889ef51ded4243784eaee0218b9563794a)